keel-core/src/main/kotlin/com/netflix/spinnaker/keel/persistence/ResourceRepository.kt

```diff
@@ -161,99 +128,6 @@ interface ResourceRepository : PeriodicallyCheckedRepository<Resource<out Resour
    */
   override fun itemsDueForCheck(minTimeSinceLastCheck: Duration, limit: Int): Collection<Resource<out ResourceSpec>>
 
-  fun getStatus(id: String): ResourceStatus {
```

Comment:
Functions moved out of repository and into `ResourceService` bean as per discussions around rethinking responsibility of repositories, in particular not having any "business logic" in them and letting them focus on data-access-related stuff.